### PR TITLE
chore: PX-5429 adding new events for BN and MO

### PR DESCRIPTION
**Pull request by Mitch-Henson**
The type of this PR is: **Feature**

This PR partially resolves [PX-5429]

### Description

This PR adds a new event to Cohesion to enable the completion of Volt work as part of [PX-5429]

### PR Checklist (tick all before merging)

- [x] If I've added a new file to the tree I've exported it from the common [`index.ts`](https://github.com/artsy/cohesion/blob/main/src/Schema/Events/index.ts)
- [x] I've added comments with examples for any new interfaces and ensured that they're in the docs
- [x] No platform-specific terminology has been used outside of `click` and `tap` (platform is inferred by the DB storing events)

[PX-5429]: https://artsyproduct.atlassian.net/browse/PX-5429?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[PX-5429]: https://artsyproduct.atlassian.net/browse/PX-5429?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Schema/Events/Click.ts [hunk 1]
```diff
@@ -1484,3 +1484,51 @@ export interface ClickedOnPriceDisplayDropdown {
...
+ *    label: ["true", "false"]
+ *  }
+ * ```
+ */
+export interface ClickedOnBuyNowCheckbox {
+  action: ActionType.clickedOnBuyNowCheckbox
+  context_module: ContextModule
+  context_page_owner_type: PageOwnerType
+  context_page_owner_id?: string
+  label: string
```
Comment (louislecluse) [inline src/Schema/Events/Click.ts:1509]: Can we do boolean?

**Comment by louislecluse** [comment]
Left 1 comment, looks otherwise, thanks!

File: src/Schema/Events/Click.ts (hunk 1 above)
Comment (Mitch-Henson) [reply src/Schema/Events/Click.ts:1509]: Absolutely!

**Review by louislecluse — APPROVED** [review]

**Review by nspinazz89 — APPROVED** [review]
LGTM 🚀

**Comment by artsyit** [comment]
:rocket: PR was released in [`v4.106.0`](https://github.com/artsy/cohesion/releases/tag/v4.106.0) :rocket:

**Pull request merged**